core/dbt/include/global_project/macros/adapters/common.sql

```diff
@@ -189,7 +189,7 @@
 
 {% macro default__information_schema_name(database) -%}
   {%- if database -%}
-    {{ adapter.quote_as_configured(database, 'database') }}.INFORMATION_SCHEMA
+    database.INFORMATION_SCHEMA
```

Comment:
Do we not want `{{ database }}` here?

Reply:
yes 🤦‍♂ today's lesson is "why we copy+paste instead of re-writing things"